config

```diff
@@ -551,6 +551,7 @@ case "$GUESSOS" in
   pentium-*-vxworks*) OUT="vxworks-pentium" ;;
   simlinux-*-vxworks*) OUT="vxworks-simlinux" ;;
   mips-*-vxworks*) OUT="vxworks-mips";;
+  e2k*) OUT="linux-generic64 -DL_ENDIAN" ;;
```

Comment:
Will only Linux be implemented for those processors, ever?  If you can predict the future that well, this is fine, but if not, the case should be `e2k-*-linux*`

Reply:
I hardly can imagine that any not Linux-based OS will appear in Russia more or less soon.

Reply:
Well, I was wrong. @levitte, many thanks!